src/gr11review/part2/Utility.java

```diff
@@ -1,8 +1,290 @@
+    public static int sumNumbers(String str) {
+        int strSum = 0;
+        String strTemp = "0";
+
+        for (int i = 0; i < str.length(); i++) {
+            // if character at i is digit, add index to strTemp
+            if (Character.isDigit(str.charAt(i))) {
+                strTemp += str.charAt(i);
+            }
+            // Once index is not a digit, convert strTemp to int and add to strSum
+            else {
+                strSum += Integer.parseInt(strTemp);
+                strTemp = "0";
+            }
+        }
+
+        // return sum including the strTemp at last index in case strTemp still has a value
+        return strSum + Integer.parseInt(strTemp);
+    }
+
+    /**
+     * FileIO1: This method returns the longest word in the file,
+     * given the name of a file filenametxt that contains a single word on each line.
+     * @param filenametxt a file that contains a single word on each line.
+     * @return returns the longest word in the file.
+     * @author L. He
+     */
+    public static String longestWord(String filenametxt) throws IOException {
+		BufferedReader thefile = new BufferedReader(new FileReader("src/gr11review/part2/" + filenametxt));
+
+        // Variables
+        String strLongestWord = "";
+        String str = "";
+        int intLongestWord = -1;    
+
+        // Finds the longest word in the file
+        while(str != null){
+            str = thefile.readLine();
+            if(str != null){
+                if(str.length() > intLongestWord) {
+                    intLongestWord = str.length();
+                    strLongestWord = str;
+                }
+            }
+        } thefile.close();
+        return strLongestWord;
+    }
+
+    /**
+     * Array1: This method returns the version of the given array where all the 10's have been removed. 
+     * @param nums array values. 
+     * @return returns the array without tens 
+     * @author L. He
+     */
+    public static int[] withoutTen(int[] nums) {
+        // Variables
+        int intCount = 0;
+
+        // Resulting Array
+        int[] finalArray = new int[nums.length];
+
+        // Calculation
+        for(int i = 0; i < nums.length; i++) {
+            if(nums[i] == 10) {
+            } else {
+                finalArray[intCount] = nums[i];
+                intCount++;
+            }
+        }
+
+        for(int i = intCount; i < nums.length; i++) {
+            finalArray[i] = 0;
+        }
+        return finalArray;
+    }
+        
+    /**
+
+     * FileIO2: This method returns the 1st word alphabetically in the given file
+     * @param filenametxt a file that contains a single word on each line.
+     * @return returns the alphabetically first word in the file.
+     * @throws IOException
+     * @author K. Huang
+     */
+    public static String alphaWord(String filenametxt) throws IOException {
+        BufferedReader file = new BufferedReader(new FileReader("src/gr11review/part2/" + filenametxt));
```

Comment:
need to account for the fullpath here.  don't add a prefix to the filepath i.e `src/gr11review/part2/`.  All for the incoming `filenametxt` value to be the full path. i.e 

`/Users/ericfabroa/ICS4U_workspace/Grade 11 Review 2 - Methods and Testing-10-30-2022-10-23-02/kevin and lydia/src/gr11review/part2/words.txt`